proto/tendermint/abci/types.proto

```diff
@@ -332,7 +331,6 @@ message ResponseFinalizeBlock {
   repeated ValidatorUpdate         validator_updates       = 3 [(gogoproto.nullable) = false];
   tendermint.types.ConsensusParams consensus_param_updates = 4;
   bytes                            app_hash                = 5;
```

Comment:
This is a subjective conversation about words so perhaps it won't mean anything, but do we want to change the terminology from `app_hash` to `data`. `app_hash` is probably what's more commonly used, but I like `data` because it doesn't assume a hash - but rather any arbitrary `data` that will be agreed upon

Reply:
I've always found `data` "hard to grasp" when working on the spec (had to think for 10 secs what `data` this was referring to).
How about `app_data` ?

Reply:
Honestly, I'm also happy with `app_hash`. I'll leave it up to your discretion